src/Symfony/Component/Security/composer.json

```diff
@@ -33,6 +33,7 @@
     "require-dev": {
         "psr/container": "^1.0",
         "symfony/finder": "~2.8|~3.0|~4.0",
+        "symfony/polyfill-ctype": "~1.8",
```

Comment:
Should also be added to the main composer.json file, right?

Reply:
it is already part of the main composer.json